Lock.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef LOCK_HPP
+#define LOCK_HPP
+
+class Lock {
+	private:
+		bool locked;
+		int pin;
+	public:
+		Lock(int pin);
+		Lock(const Lock&) = delete;
+		Lock& operator=(const Lock&) = delete;
+
+		void init();
+		void unlock(int timeBeforeLock = 0);
+		void lock();
```

Comment:
I think in this PR we should also call these classes and methods inside the `.ino` file